trunk/SUAVE/Analyses/Stability/AVL.py

```diff
@@ -0,0 +1,440 @@
+import SUAVE
+from SUAVE.Core import Units, Data
+from SUAVE.Core import redirect
+
+from SUAVE.Analyses.Mission.Segments.Conditions.Aerodynamics import Aerodynamics
+from SUAVE.Analyses.Mission.Segments.Conditions.Conditions   import Conditions
+
+from SUAVE.Methods.Aerodynamics.AVL.write_geometry   import write_geometry
+from SUAVE.Methods.Aerodynamics.AVL.write_run_cases  import write_run_cases
+from SUAVE.Methods.Aerodynamics.AVL.write_input_deck import write_input_deck
+from SUAVE.Methods.Aerodynamics.AVL.run_analysis     import run_analysis
+from SUAVE.Methods.Aerodynamics.AVL.translate_data   import translate_conditions_to_cases, translate_results_to_conditions
+from SUAVE.Methods.Aerodynamics.AVL.purge_files      import purge_files
+from SUAVE.Methods.Aerodynamics.AVL.Data.Results     import Results
+from SUAVE.Methods.Aerodynamics.AVL.Data.Settings    import Settings
+from SUAVE.Methods.Aerodynamics.AVL.Data.Cases       import Run_Case
+
+# local imports
+from Stability import Stability
+
+# import SUAVE methods
+from SUAVE.Methods.Flight_Dynamics.Static_Stability.Approximations.Tube_Wing.taw_cmalpha import taw_cmalpha
+from SUAVE.Methods.Flight_Dynamics.Static_Stability.Approximations.Tube_Wing.taw_cnbeta import taw_cnbeta
+from SUAVE.Methods.Flight_Dynamics.Static_Stability.Approximations.datcom import datcom
+from SUAVE.Methods.Flight_Dynamics.Dynamic_Stability import Approximations as Approximations
+from SUAVE.Methods.Flight_Dynamics.Dynamic_Stability import Full_Linearized_Equations as Full_Linearized_Equations
+from SUAVE.Methods.Flight_Dynamics.Dynamic_Stability.Full_Linearized_Equations import Supporting_Functions as Supporting_Functions
+
+# Package imports
+import time
+import pylab as plt
+import os
+import numpy as np
+import sys
+import sklearn
+from sklearn import gaussian_process
+from sklearn import neighbors
+from sklearn import svm
+from shutil import rmtree
+from warnings import warn
+
+## ----------------------------------------------------------------------
+##  Class
+## ----------------------------------------------------------------------
+
+class AVL(Stability):
+    """ SUAVE.Analyses.Aerodynamics.AVL
+        aerodynamic model that performs a vortex lattice analysis using AVL
+        (Athena Vortex Lattice, by Mark Drela of MIT).
+
+        this class is callable, see self.__call__
+
+    """
+
+    def __defaults__(self):
+        self.tag        = 'avl'
+        self.keep_files = True
+
+        self.settings = Settings()
+
+        self.current_status = Data()
+        self.current_status.batch_index = 0
+        self.current_status.batch_file  = None
+        self.current_status.deck_file   = None
+        self.current_status.cases       = None
+        
+	self.settings.filenames.log_filename = sys.stdout
+        self.settings.filenames.err_filename = sys.stderr
+        
+        # Conditions table, used for surrogate model training
+        self.training = Data()        
+	
+	#HALE UAV Cruise
```

Comment:
Take out these defaults.

You should set your project specific defaults in your Analysis script.